src/dagmc.cpp

```diff
@@ -461,10 +459,14 @@ std::string DAGUniverse::get_uwuw_materials_xml() const
   ss << "</materials>";
 
   return ss.str();
+#else
+  throw std::runtime_error("DAGMC was not configured with UWUW.");
```

Comment:
I think the general guidance is to use `fatal_error` here rather than throwing an exception, although with the python side now handling messages originating from exceptions, it doesn't matter a lot

Reply:
Makes sense. I've updated these to use `fatal_error()`.